crypto/benchmark_test.go

```diff
@@ -0,0 +1,41 @@
+package crypto
+
+import (
+	"testing"
+
+	"github.com/CoderZhi/go-ethereum/crypto"
```

Comment:
coderzhi?

Reply:
go-ethereum is imported by coderzhi when evm is introduced.